### ISSUE TITLE
Eslint Test on Combobox fulfill ARIA requirement

### ISSUE THREAD

**Issue by jonasdd** [jsx-eslint/eslint-plugin-jsx-a11y#789]
When working on a combobox Eslint throws an error about missing aria-controls attribute on combobox role div. Pattern used as follows : 

```html
<div role="combobox"
            aria-expanded="false"
            aria-owns="ex2-listbox"
            aria-haspopup="listbox"
            id="ex2-combobox">
        <input type="text"
            aria-autocomplete="list"
            aria-readonly="true"
            aria-controls="ex2-listbox"
            aria-activedescendant="exp_elem_Np"
            aria-labelledby="ex2-label"
            id="ex2-input">
    </div>
    <ul aria-labelledby="ex2-label"
        tabindex="-1"
        role="listbox"
        id="ex2-listbox"
        class="listbox hidden">
    </ul>
```

According to [ARIA 1.1](https://www.w3.org/TR/wai-aria-1.1/#combobox) requirement, aria-controls is required on the **input** not on the element with **combobox** role.

```html
<div aria-label="Tag" role="combobox" aria-expanded="true" aria-owns="owned_listbox" aria-haspopup="listbox">
    <input type="text" aria-autocomplete="list" aria-controls="owned_listbox" aria-activedescendant="selected_option">
</div>
<ul role="listbox" id="owned_listbox">
    <li role="option">Zebra</li>
    <li role="option" id="selected_option">Zoom</li>
</ul>
```

Also, the element that has a combobox role element should have an **aria-owns** attribute instead of the aria-controls element requested by the Eslint test. 
Finally, it seems to be important that element with aria-ows attr is a sibling of the list box (here, the <ul> element).
